.github/workflows/build-images.yml

```diff
@@ -363,6 +363,15 @@ jobs:
         else
           make docker-build-operator
         fi
+        # Check if image exists in private repo, if not exist, push to repo
```

Comment:
Have e2e tests run at this step? Before push image to private repo, we need to wait for all e2e tests to be done successfully.

Reply:
No, this is build step for the e2e test. I thought for github one we just want to push if the image is not there. For the private repo(Opentext one) we want to check for e2e test result. Is that correct, or we want to check for both